### DISCUSSION TITLE
Expand navbar menu when submenu is active

### DISCUSSION THREAD

**Discussion by ibalat** [livewire/flux#785] [Feature Requests]
My sidebar have many navbar item, so all of them have `expandable :expanded="false"`. But when I visit a submenu, its parent navbar is closing and we don't see current submenu. How can we auto expand parent navbar if its submenu is active?

![Image from Photopea](https://github.com/user-attachments/assets/cc8f1f4e-f8d9-42dd-b811-9eb376d03106)

**Comment by jeffchown** [comment]
@ibalat Maybe change it to `:expanded="isSubmenuActive()"` where the `isSubmenuActive()` method checks the current url to see if it is one of the urls from a submenu item?

**Reply by ibalat** [reply]
flux already has "current" checking operation. I don't want to check again it because my navbars have so much submenu.

**Comment by joshhanley** [comment]
@ibalat this sounds more like a bug. Can you open an issue and include all these details there? Please include a code snippet we can copy and paste into our own apps with steps to reproduce. Please include any Blade variable definitions that are used and use a little code as possible to reproduce the issue.

Going to close this in favour of that. Thanks!

**Reply by ibalat** [reply]
I have opened the issue, thanks for now: https://github.com/livewire/flux/issues/789
